src/ui-interaction/textEditorView.ts

```diff
@@ -3,7 +3,8 @@ import { executeQuickPick } from './commandPrompt';
 import { Duration, log, openFile, pause } from '../core/miscellaneous';
 import { getBrowser } from './workbench';
 import { retryOperation } from '../retryUtils';
-import fs from 'fs';
+import * as fs from 'node:fs/promises';
```

Comment:
I see in visualforceUtils.ts we're importing just 'fs/promises', I know it's the same but let's just use the same style for consistency!